### PR TITLE
Validator: whitelist allow-ssr-img for amp-pixel

### PR DESCRIPTION
**Pull request by keithwrightbos**
See #12425 for background.  Used in AMP4ADS creatives as part of SSR transformation indicating that img tag can be directly inserted within amp-pixel (essentially indicates no macros are present).

### CODE REVIEW

File: validator/testdata/amp4ads_feature_tests/amp_pixel_ssr.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<!--
+  Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (honeybadgerdontcare) [inline validator/testdata/amp4ads_feature_tests/amp_pixel_ssr.html]: 2015/2017

File: validator/testdata/amp4ads_feature_tests/amp_pixel_ssr.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS-IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the license.
+-->
+<!--
+  Test Description:
+  This is the minimum valid A4A document; it includes all of the required
+  fields from the spec, but nothing else.
```
Comment (honeybadgerdontcare) [inline validator/testdata/amp4ads_feature_tests/amp_pixel_ssr.html]: please update comment

File: validator/validator-main.protoascii [hunk 3]
```diff
@@ -3560,6 +3560,7 @@ tags: {  # <amp-pixel>
     name: "referrerpolicy"
     value: "no-referrer"
   }
+  attrs: { name: "allow-ssr-img" }
```
Comment (honeybadgerdontcare) [inline validator/validator-main.protoascii]: please alphabetize

File: validator/testdata/amp4ads_feature_tests/amp_pixel_ssr.html (hunk 1 above)
Comment (keithwrightbos) [reply validator/testdata/amp4ads_feature_tests/amp_pixel_ssr.html]: Done

File: validator/testdata/amp4ads_feature_tests/amp_pixel_ssr.html (hunk 2 above)
Comment (keithwrightbos) [reply validator/testdata/amp4ads_feature_tests/amp_pixel_ssr.html]: Done

**Pull request merged**